docs/source/cli/poet.rst

```diff
@@ -19,23 +19,41 @@
 PoET CLI
 ********
 
+The PoET CLI initializes the proof-of-concept consensus mechanism for
+Sawtooth by generating enclave setup information and creating a Batch
+for the genesis block. For more information, see :doc:/architecture/poet.
```

Comment:
Sorry - I gave you bad RST code in my first review.  It needs backticks:
     :doc:`/architecture/poet`

If this doesn't work, copy the doc rev in app_developers_guide/ubuntu.rst.